### PR TITLE
changes

### PR DESCRIPTION
**Pull request by avinash70008**

### CODE REVIEW

File: src/components/Pages/JoditEditor/cloudinaryService.js
Comment (durgeshgupta15) [inline src/components/Pages/JoditEditor/cloudinaryService.js:1]: why did you create this new file. there is already a file apiService in which there is code available to file upload. use that remove this file. function name -> uploadingImage

File: src/components/BlogMainComponents/BlogMain.jsx
Comment (durgeshgupta15) [inline src/components/BlogMainComponents/BlogMain.jsx:1]: why did you use this BlogMain.jsx file. code is already there in BlogEditor.js. check that and remove this folder BlogMainComponenets

File: src/components/Pages/JoditEditor/cloudinaryService.js
Comment (avinash70008) [reply src/components/Pages/JoditEditor/cloudinaryService.js:1]: its already been deleted and understand the flow of code

File: src/components/BlogMainComponents/BlogMain.jsx
Comment (avinash70008) [reply src/components/BlogMainComponents/BlogMain.jsx:1]: its also deleted

**Pull request merged**